Main.java

```diff
@@ -1,21 +1,41 @@
 class Main {
   public static void main(String[] args) {
-   // kindly use appropriate data types for your declaration
    // declare variables days the debtor defaulted, amount the debtor is to pay per day,amount the debtor paid and assign values to them
+   int daysdefaulted =16;
+  int amountperday =1000;
+  int amounttopay;
    
    //Declare the variable of the total amount the debtor is to pay, the days the debtor paid for, amount the debtor is left to pay, days the debtor has not payed for
-
+   int dayspaidfor;
+   int amountlefttopay;
+   int daysnotpaid;
+   int amountpaid =2000;
+  int amountnotinday;
```

Comment:
Good job but your amount should have been in float, also you should have used camelCase naming method